x-pack/test/cloud_security_posture_functional/page_objects/csp_dashboard_page.ts

```diff
@@ -110,12 +114,11 @@ export function CspDashboardPageProvider({ getService, getPageObjects }: FtrProv
     },
 
     getKubernetesComplianceScore: async () => {
-      await dashboard.getKubernetesSummarySection();
-      return await testSubjects.find('dashboard-summary-section-compliance-score');
-    },
+      await retry.waitFor(
```

Comment:
for some unknown reason, this getter wasn't resolving in 10s timeframe and the test was failing when running on CI (and local when running with `yarn test:ftr` and not with server and runner separately)